_posts/2018-11-19-open-core-engineering-group-meeting.md

```diff
@@ -0,0 +1,84 @@
+
+Furthermore a discussion takes place about whether the design governance or process increase the difficulty of working in the compiler (with different opinions).
+
+For language design certain statements are now fully official:
+- The decision on TCs is clear: we won't put a fully-fledged version of TCs into F# while there is any chance that the corresponding feature will be put into C# (quote Don Syme)
+- we will seek to evolve the SRTP mecahnism to accurately support some TC-like scenarios but in a fairly conservative way (quote Don Syme)
+
+Further question points which have been adressed afterwards are:
+- separation of compiler and tools into different repositories
+- democratic voted group of “core compiler engineers” to the so called “Compiler Group”
+- regularly timed open chat / discussion with the “Compiler Group” about feature implementation details + compiler architecture design
+- List of people with their knowledge in the compiler
+- Ask people that have a deep knowledge of f# compiler what needs to be done. Looking at github a first list of people to ask could be: Kevin Ransom, Don Syme, Steffen Forkmann, Vasily Kirichenko
+- Provide a beta channel promoted by Microsoft which allows to easily download a custom F# compiler version with a specific feature in beta version
+
+Don Syme agrees on that the various F# compiler repositories should be merged in the future to less confuse people. Phillip Carter added that they are still working on improving the infrastructure of the visualfsharp repository to improve CI times.
+
+Additionaly Don Syme expressed again, that he's much in favour of regular timed compiler related discussions either held by him or a group of volunteers.
+
+Regarding Beta Channels Don Syme gave over to Phillip Carter who tells that the Infrastructure for the .NET SDK isn't ready for such yet. No further particular decision or statement was given for this proposal.
+
+After 1 1/2 hours the time for the meeting was up and the discussion ended.
+
+### Participating members:
+
+- Don Syme
+- Jindřich Ivánek
+- Krzysztof Cieslak
+- Roman Sachse
+- Toby Shaw
+- Isaac Abraham
+- Phillip Carter
+- Chet Husk
+- Mark Lambert
+- Bjørn Bæverfjord
+- robert kuzelj
+- Dave Thomas
+- Oskar Gewalli
+- Thomas Boby
+- Bryant Huang
+- Gerard
+- Elliott V. Brown
+- Diego Esmerio
+- Victor Peter Rouven Müller
+
+### Proposed core engineering tasks:
```

Comment:
We should think about how we track these.  We can discuss this next Monday

Reply:
I agree on this.